modules/xmpp/ChatRoom.js

```diff
@@ -447,6 +456,11 @@ export default class ChatRoom extends Listenable {
             if (member.displayName) {
                 memberOfThis.displayName = member.displayName;
             }
+
+            if (memberOfThis.status !== member.status) {
```

Comment:
Do you think it's worthwhile to leave a comment about this or would it be obvious to someone who is working in this code?